csharp/src/Microsoft.AI.MachineLearning/Microsoft.AI.MachineLearning.Rules.Project.xml

```diff
@@ -8,13 +8,13 @@
     <DataSource Persistence="ProjectFile" HasConfigurationCondition="false" Label="Globals" />
   </Rule.DataSource>
 
-  <StringProperty Name="OnnxRuntimePathOverride"
-                  DisplayName="Custom ONNXRuntime DLL"
-                  Description="Specifies a path to a custom OnnxRuntime dll."
-                  Category="General" />
+  <BoolProperty Name="DisableOnnxRuntimeDllCopy"
+                DisplayName="Disable onnxruntime.dll copy on build"
+                Description="Suppresses binplacing the onnxruntime.dll from the Mirosoft.AI.MachineLearning NuGet distribution."
```

Comment:
>onnxruntime [](start = 55, length = 11)

In what scenario would onnxruntime.dll not be binplaced? Doesn't Microsoft.AI.MachineLearning.dll depend on onnxruntime.dll?

Reply:
When the app developer wants to binplace another one.